content/200-concepts/100-components/02-prisma-client/260-client-extensions/130-result.mdx

```diff
@@ -9,8 +9,113 @@ tocDepth: 3
 
 <Admonition type="warning">
 
-We plan to add `result` extensions to a future version of Prisma. This page is a placeholder.
+This page is in draft.
 
 </Admonition>
 
+From version 4.7.0, you can use `result` [Prisma Client extensions](/concepts/components/prisma-client/client-extensions) to add custom fields and methods to query results.
+
+Use the `$extends` [client-level method](/reference/api-reference/prisma-client-reference#client-methods) to create an _extended client_. An extended client is a forked variant of the standard Prisma Client with an extension applied. Use the `results` extension type to add custom objects to query results. ** to models in your schema? **
```

Comment:
```suggestion
Use the `$extends` [client-level method](/reference/api-reference/prisma-client-reference#client-methods) to create an _extended client_. An extended client is a forked variant of the standard Prisma Client with an extension applied. Use the `results` extension type to add custom properties to query results.
```

Reply:
"fields" also works

Reply:
We should go with `fields` to be consistent with other docs when we are talking about fields. But here we mean fields or methods. I think I need to change the language in the rest of the PR to say "fields" instead of "properties", but here we need to say "fields or methods". So:

```suggestion
Use the `$extends` [client-level method](/reference/api-reference/prisma-client-reference#client-methods) to create an _extended client_. An extended client is a forked variant of the standard Prisma Client with an extension applied. Use the `results` extension type to add custom fields and methods to query results. 
```